src/Makefile.in

```diff
@@ -97,9 +97,9 @@ PFLAGS_EXTRA   += @PFLAGS_EXTRA@
 # - Note that fpc.cfg already defines -vinw, so add -v0 first
 # - The stack check (-Ct) might not work with enabled threading
 # - Do we need -Coi?
-PFLAGS_BASE_DEFAULT    := -Si -Sg- -Sc- -v0Binwe
-PFLAGS_DEBUG_DEFAULT   := -Xs- -g -gl -dDEBUG_MODE
-PFLAGS_RELEASE_DEFAULT := -Xs- -O2
+PFLAGS_BASE_DEFAULT    := -Si -Sg- -Sc- -v0Binwe -gl
+PFLAGS_DEBUG_DEFAULT   := -Xs- -g -dDEBUG_MODE
+PFLAGS_RELEASE_DEFAULT := -Xs- -O2 -OoNOSTACKFRAME
```

Comment:
flatpak-builder already passes -g in CFLAGS and CXXFLAGS. We could convert that GCC flag to flags for FPC instead of removing the possibility to make a build without debug info.

Reply:
I see there's https://github.com/flathub/eu.usdx.UltraStarDeluxe/pull/23 since the comment was made, but I have no idea which of these can be merged now. Or do I just merge all three (flatpak 23, this one, and 1007), close flatpak 22 and It'll Work ?

I can't tell if flatpak 23 will handle both the before-this-PR and after-this-PR makefile, since it appears to patch it (which is fine, keep the weird behaviour away from everything else) but if you say it's fine then I'll take your word for it.

Reply:
> I can't tell if flatpak 23 will handle both the before-this-PR and after-this-PR makefile, since it appears to patch it (which is fine, keep the weird behaviour away from everything else) but if you say it's fine then I'll take your word for it.

The patch would have to be revised. But is debug.patch really weird behavior? In my opinion using Free Pascal is weird and converting the common CFLAGS variable to flags for our weird compiler reduces the weirdness.

Reply:
Yeah, "weird" maybe wasn't the best wording. I'm actually okay if you want to do something about CFLAGS behaviour in the makefile in the future, but at this point I just want to get these merged first.

The way I'm reading it, I should merge flatpak 23 first, since it's based on the current makefile? then once that's done, merge 1019 and 1007, at which point we'd need a new flatpak PR, but it'll be very small and it'll fail until we can actually update the tag/commit as well, and only then merge it?